plugin/src/automation/lift/liftconfig.ml

```diff
@@ -402,14 +455,46 @@ let lift_constr env sigma c trm =
      else
        (* inductive case - refold *)
        refold l env (lift_to l) app rec_args sigma
+  | UnpackSigma ->
+     (* Delta-recude the function, and simplify projections of existentials *)
```

Comment:
This needs cleaning and more of an explanation of why we do it (mostly to make things look much nicer)